aiconfig-docs/docs/gradio-notebook.md

```diff
@@ -144,15 +144,42 @@ model_output = await config.run('prompt_1')
 
 ## Supported Models
 
-- **Gradio Notebooks support models which use the [Hugging Face Inference API](https://huggingface.co/docs/api-inference/index).** [Hugging Face Tasks](https://huggingface.co/tasks) that are supported:
+### Inference API
+**Gradio Notebooks support models which use the [Hugging Face Inference API](https://huggingface.co/docs/api-inference/index).** [Hugging Face Tasks](https://huggingface.co/tasks) that are supported:
   - [Text Generation](https://huggingface.co/tasks/text-generation)
   - [Summarization](https://huggingface.co/tasks/summarization)
   - [Translation](https://huggingface.co/tasks/translation)
   - [Automatic Speech Recognition (ASR)](https://huggingface.co/tasks/automatic-speech-recognition)
   - [Text-to-Speech](https://huggingface.co/tasks/text-to-speech)
   - [Text-to-Image](https://huggingface.co/tasks/text-to-image)
   - [Image-to-Text](https://huggingface.co/tasks/image-to-text)
-- **Want to access other models?** Model parsers exist for local models, add them to the `ModelParserRegistry` in `app.py`. Have questions? Chat with us on [Discord](https://discord.com/invite/xBhNKTetGx).
+
+### Local Models
+Model parsers also exist for local models associated with most of the above tasks (via Hugging Face Transformers and Diffusers library). 
+
+NOTE: Using local models will download the models to your space, using up space resources, even if the space user is not an owner of the space. Downloading the models will also require a significant wait when running a cell if they have not already been downloaded to your space. Please be aware of these considerations when using local models.
```

Comment:
Prefer to use `> [!CAUTION]  ` instead: https://github.com/orgs/community/discussions/16925
<img width="952" alt="Screenshot 2024-02-13 at 18 43 25" src="https://github.com/lastmile-ai/aiconfig/assets/151060367/f5c92304-d913-442a-bf29-062d59b95e49">